common/DbHandler.py

```diff
@@ -1,73 +1,57 @@
-# NOTE: This class is just a prototype and it's just for testing purposes
-
-from werkzeug.security import safe_str_cmp
+# NOTE: This class is for handling all calls to database
 
 from common.Link import Link
 from common.User import User
 
-links = [
-    Link("www.stackoverflow.com", ["programming"]),
-    Link("www.geeksforgeeks.com", ["programming", "learning"])
-]
-
-USERS = {
-    'user1': User(1, 'user1', 'zanjan', links[0]),
-    'test': User(2, 'test', 'test', links)
-}
+from db import db
 
 
 class DbHandler():
+    @staticmethod
+    def get_user_object(username: str):
+        return User.query.filter_by(username=username).first()
+
     @staticmethod
     def validate_login(username: str, password: str):
-        user = USERS.get(username)
+        user = User.query.filter_by(username=username).first()
 
-        return_message = ""
-        if user and safe_str_cmp(
-            user.password.encode('utf-8'), password.encode('utf-8')
-        ):
-            return_message = user
+        # Check if user exists and password is correct
+        if user and user.check_password(password):
+            return user
         else:
-            return_message = None
-
-        return return_message
+            return None
 
     @staticmethod
     def add_new_user(username: str, password: str):
         # Check if user exists
-        if username in USERS:
-            return 1
+        user = User.query.filter_by(username=username).first()
+        if user:
+            return "USER_EXISTS"
 
-        # If user doesn't exist, get Id for it and Signup
-        max_id = 0
-        for user in USERS.values():
-            if user.id > max_id:
-                max_id = user.id
+        user = User(username=username, password_hash=password)
+        db.session.add(user)
+        db.session.commit()
 
-        # Add new user
-        USERS[username] = User(max_id+1, username, password)
-
-        return 0
-
-    @staticmethod
-    def get_links(username: str):
-        return USERS.get(username).links
+        return "OK"
 
     @staticmethod
-    def append_new_link(username: str, new_link: Link):
-        USERS[username].append_new_link(new_link)
-        return 0
+    def append_new_link(new_link: Link):
```

Comment:
It would be great to determine the return type of function and other functions as well:
```suggestion
    def append_new_link(new_link: Link) -> str:
```